.travis.yml

```diff
@@ -8,7 +8,6 @@ rvm:
 script:
   - 'bundle exec rails db:migrate RAILS_ENV=test'
   - 'bundle exec rspec spec'
-  - 'bundle exec rubocop'
```

Comment:
because we have CodeClimate now. I turned it to be required for merge in settings